config/routes.js

```diff
@@ -66,6 +66,7 @@ module.exports.routes = {
   'GET    /noo/community/:communityId':                   'CommunityController.findOne',
   'POST   /noo/community/:communityId':                   'CommunityController.update',
   'GET    /noo/community/:communityId/settings':          'CommunityController.findSettings',
+  'GET    /noo/community/:communityId/settings/slack':    'CommunityController.addSlack',
```

Comment:
I think I was trying it with a post, wasn't working, switched it to get and then changed something else and then it was working. but I think the thing that fixed it wasn't switching from Post to Get, so we could switch it back. Not much need though, this way is equally supported https://api.slack.com/web#basics